packages/angular/stories/forms.stories.ts

```diff
@@ -0,0 +1,363 @@
+import type { Meta, StoryFn } from '@storybook/angular';
+import '@sl-design-system/button/register.js';
+import '@sl-design-system/button-bar/register.js';
+import '@sl-design-system/form/register.js';
+import '@sl-design-system/inline-message/register.js';
+import type { AbstractControl, ValidationErrors } from '@angular/forms';
+import { FormControl, FormGroup, FormsModule, ReactiveFormsModule } from '@angular/forms';
+import type { ElementRef } from '@angular/core';
+import { CUSTOM_ELEMENTS_SCHEMA, Component, ViewChild } from '@angular/core';
+import type { Form } from '@sl-design-system/form';
+import { moduleMetadata } from '@storybook/angular';
+import { FormsModule as CoreFormsModule } from '../src/forms/forms.module';
+
+@Component({
+  selector: 'sla-all-form-controls-reactive',
+  template: `
+    <sl-form [formGroup]="formGroup">
+      <sl-form-field label="Text field">
+        <sl-text-field formControlName="textField"></sl-text-field>
+      </sl-form-field>
+
+      <sl-form-field label="Textarea">
+        <sl-textarea formControlName="textarea"></sl-textarea>
+      </sl-form-field>
+
+      <sl-form-field label="Checkbox">
+        <sl-checkbox formControlName="checkbox" value="checked">Checkbox</sl-checkbox>
+      </sl-form-field>
+
+      <sl-form-field label="Select">
+        <sl-select formControlName="select">
+          <sl-select-option value="1">Option 1</sl-select-option>
+          <sl-select-option value="2">Option 2</sl-select-option>
+          <sl-select-option value="3">Option 3</sl-select-option>
+        </sl-select>
+      </sl-form-field>
+
+      <sl-form-field label="Switch">
+        <sl-switch formControlName="switch" reverse value="toggled">Toggle me</sl-switch>
+      </sl-form-field>
+
+      <sl-form-field label="Checkbox group">
+        <sl-checkbox-group formControlName="checkboxGroup">
+          <sl-checkbox value="0">One</sl-checkbox>
+          <sl-checkbox value="1">Two</sl-checkbox>
+          <sl-checkbox value="2">Three</sl-checkbox>
+        </sl-checkbox-group>
+      </sl-form-field>
+
+      <sl-form-field label="Radio group">
+        <sl-radio-group formControlName="radioGroup">
+          <sl-radio value="1">One</sl-radio>
+          <sl-radio value="2">Two</sl-radio>
+          <sl-radio value="3">Three</sl-radio>
+        </sl-radio-group>
+      </sl-form-field>
+    </sl-form>
+
+    <pre>{{ formGroup.value | json }}</pre>
+  `
+})
+export class AllFormControlsReactiveComponent {
+  formGroup = new FormGroup({
+    textField: new FormControl('Text field'),
+    textarea: new FormControl('Textarea'),
+    checkbox: new FormControl('checked'),
+    select: new FormControl('1'),
+    switch: new FormControl('toggled'),
```

Comment:
This starts with a state `toggled` but it when you click it it goes to `false` and when you click it again it becomes `true`, this is inconsistent.

Reply:
Fixed